bin/vitamin.js

```diff
@@ -6,15 +6,17 @@ import path from 'path';
 import rimraf from 'rimraf';
 import { spawn } from 'child_process';
 import fs from 'fs';
+import { spawn as npmRunSpawn } from 'npm-run';
```

Comment:
Well, I wasn't a big fan of using a `test` script inside the `package.json` of vitaminjs, because, in this context, we are not using it for testing vitaminjs but for testing the application itself. 
But, now that you're pointing it, I realised that we could use the `yarn jest` command, that comes directly with yarn, without needing to modify the `package.json`, and with no extra dependencies.

What do you think, @LucasPerso @christophehurpeau ?